spec/spec_helper.rb

```diff
@@ -132,6 +132,10 @@
   # include spec utils
   config.include SpecUtils
 
+  config.before(:suite) do
+    Flipper.disable(:active_storage_s3)
+  end
```

Comment:
Add spec for both cases instead of globally disabling the spec

Reply:
My bad, this popped up from local testing. Added extensive test cases for both  the cases 👍